mava/advanced_usage/ff_ippo_store_experience.py

```diff
@@ -548,11 +548,9 @@ def run_experiment(_config: DictConfig) -> None:
     def _reshape_experience(experience: Dict[str, chex.Array]) -> Dict[str, chex.Array]:
         """Reshape experience to match buffer."""
         # Swap the T and NE axes (D, NU, UB, T, NE, ...) -> (D, NU, UB, NE, T, ...)
-        experience: Dict[str, chex.Array] = tree.map(lambda x: x.swapaxes(3, 4), experience)
```

Comment:
I really like that we can take this away. What did you change?

Reply:
It was one of the mypy rules that I felt was a bit strict and also wasn't a default